crates/wast/src/core/custom.rs

```diff
@@ -368,6 +369,8 @@ fn parse_sym_flags(parser: Parser<'_>) -> Result<u32> {
             "exported" = 1 << 5,
             "explicit-name" = 1 << 6,
             "no-strip" = 1 << 7,
+            "tls" = 1 << 8,
+            "absolute" = 1 << 9,
```

Comment:
Since the `wast` crate doesn't depend on `wasmparser`, I'm assuming there's no way to avoid the code duplication here?

Reply:
Unfortunately, yeah, but that's ok for now